mobile/src/main/res/values-mdpi/style.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="openHAB.OutlineButton" parent="@style/Widget.MaterialComponents.Button.OutlinedButton">
+        <item name="strokeWidth">2dp</item>
```

Comment:
mdpi is inherited by higher densities if there is no explicit specification for those. I'd probably add this definition in the base styles, make it use a dimension value and set that to 2dp in the base value and 1dp in hdpi.

Reply:
TIL about the inheritance here.